timezone/src/include/timezone/dialogs.rb

```diff
@@ -333,7 +334,7 @@ def SetTimeDialog
                     )
                   )
                 ),
-                VSpacing(1),
+                VSpacing(UI.TextMode ? 0.5 : 1),
```

Comment:
Using a floating point in text mode technically does not make sense, either use `VSpacing(UI.TextMode ? 0 : 1)` or `VSpacing(0.5)`. Or did I overlook something?

Reply:
No, you didn't, you are completely right.